README.md

```diff
@@ -179,6 +179,10 @@ A Linux or MacOS environment is required. Windows is not supported and the npm c
 3. run `npm run dev` to start the webserver, open up
    http://localhost:3001/#search and try out the basic features
 
+Before merging a branch, create a commit that updates the package's version number in `package.json`.
```

Comment:
I don't quite understand - does the version change need to be a separate commit for some reason? We usually just update the same version in the same PR that makes other changes, which may be 1 or more commits.

Reply:
Committing the version separately is convenient when you need to cherry-pick implementation. Then you don't have to deal with unrelated changes that likely produce merge conflicts.
Same for README updates.

Would you like me to rephrase this?

Reply:
Something like this would make it clearer to me:

"Before merging to master update the version in `package.json`, preferably in a separate commit."

What do you think?

Reply:
Works for me.
Perhaps we continue the conversation in the other thread, about `package-lock` at
https://github.com/LoopFiftyFour/JS-Connector/pull/53#discussion_r1778384028